database/placement_prep/job-portals.json

```
@@ -47,5 +47,12 @@
     "url": "https://monster.com",
     "category": "placement-prep",
     "subcategory": "job-portals"
+  },
+  {
+    "name": "Naukri",
+    "description": "It's a platform that helps you get the job and place the job.",
```

[proofreading]: there was a part of the description that was unclear. Consider describing the source in the suggestion below
```suggestion
    "description": "It's a platform that helps you find and get the job.",
```